topo_processor/metadata/item.py

```diff
@@ -19,6 +19,7 @@ def __init__(self, path: str, collection: Collection):
         self.collection = collection
         self.stac_item = stac.Item(
             # TODO edit these item fields, add collection back to the constuctor:stac_item after collection.stac_collection is implemented
+            stac_extensions=["checksum"],
```

Comment:
This should be added in the checksum loader section

Reply:
good point